From d1d8408c9a7c7b5ffa30371f246cc18085e11313 Mon Sep 17 00:00:00 2001
From: Doan Tran Cong Danh <congdanhqx@gmail.com>
Date: Thu, 3 Oct 2019 13:00:27 +0700
Subject: [PATCH 6/9] musl fix

Including <linux/if_bridge.h> causes the inclusion of <linux/in6.h>,
which defines 'struct in6_addr', already defined in <netinet/in.h>.
Remove <linux/if_bridge.h> to fix musl build.
---
 tools/bneptest.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tools/bneptest.c b/tools/bneptest.c
index 8bc7596b6..6b3956cfa 100644
--- a/tools/bneptest.c
+++ b/tools/bneptest.c
@@ -37,7 +37,6 @@
 #include <net/if.h>
 #include <linux/sockios.h>
 #include <netinet/in.h>
-#include <linux/if_bridge.h>
 
 #include <glib.h>
 
@@ -51,6 +50,9 @@
 #include "lib/bnep.h"
 #include "profiles/network/bnep.h"
 
+/* From <linux/if_bridge.h> */
+#define BRCTL_SET_BRIDGE_FORWARD_DELAY 8
+
 enum {
 	MODE_LISTEN,
 	MODE_CONNECT,
-- 
2.23.0

